www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -84,4 +90,284 @@ function borg_project_metrics_cron() {
       }
     }
   }
+
+  // Get and store backdrop_project_metrics top level data weekly.
+  if (date('G') == 24) {
+    if (date('l') == 'Monday') {
+      try {
+        $last_date = _get_display_metric_from_db('commits')['date'];
+        _parse_metrics('commits', $last_date);
+      }
+      catch(Exception $e) {
+        $message = $e->getMessage();
+        watchdog('borg_project_metrics', $message, array(), WATCHDOG_ERROR);
+      }
+    }
+
+    if (date('l') == 'Tuesday') {
+      try {
+        $last_date = _get_display_metric_from_db('comments')['date'];
+        _parse_metrics('comments', $last_date);
+      }
+      catch(Exception $e) {
+        $message = $e->getMessage();
+        watchdog('borg_project_metrics', $message, array(), WATCHDOG_ERROR);
+      }
+    }
+  }
+}
+
+/**
+ * Get the number of commits to Backdrop since 15 January 2015.
```

Comment:
@jenlampton this is the earliest commit I found:
  * 1688aa076163327fe0305af89072a39488ba3c36
  * 2013-09-01T05:23:28Z
  * https://api.github.com/repos/backdrop/backdrop/commits?since=2013-01-01T00:00:00Z&page=110

Do you know of anything earlier?